### PR TITLE
review done

### PR DESCRIPTION
**Pull request by NurMazumder**
users can now review games, need to clean up styling

### CODE REVIEW

File: routes/api/gamestest.js [hunk 1]
```diff
@@ -165,4 +167,95 @@ router.get("/search/platform/:search", async (req, res) => {
...
+    }
+    res.json(game.reviews);
+  } catch (err) {
+    console.error(err.message);
+    res.status(500).send('Server error');
+  }
+});
+
+
+// @route   POST api/games/review/:id
```
Comment (KevinZheng0701) [inline routes/api/gamestest.js:196]: duplicate comment

File: routes/api/gamestest.js [hunk 2]
```diff
@@ -165,4 +167,95 @@ router.get("/search/platform/:search", async (req, res) => {
...
+    const game = await Game.findOne({ rawg_id: req.params.id });
+    if (!game) {
+      return res.status(404).json({ msg: "Video game not found" });
+    }
+    const review = await Review.findById(req.params.review_id);
+    if (!review) {
+      return res.status(404).json({ msg: "Review does not exist" });
+    }
+    const user = await User.findById(req.user.id);
+    if (review.author.toString() !== req.user.id && user.role !== "admin") {
```
Comment (KevinZheng0701) [inline routes/api/gamestest.js:242]: Here you are checking the author of the review. Just curious, is the author a name or an id. If it's id then this should be fine.

File: routes/api/gamestest.js [hunk 3]
```diff
@@ -165,4 +167,95 @@ router.get("/search/platform/:search", async (req, res) => {
...
+      return res.status(401).json({ msg: "User not authorized" });
+    }
+    await review.deleteOne();
```
Comment (KevinZheng0701) [inline routes/api/gamestest.js:245]: Here you are using detete one and then filtering the games without the deleted review ID. Does node and express have a simpler delete function? That way you don't need to filter through the games.

**Review by KevinZheng0701 — COMMENTED** [review]
Looking good. Added a few comments. If they are not a problem feel free to merge.

File: routes/api/gamestest.js (hunk 2 above)
Comment (NurMazumder) [reply routes/api/gamestest.js:242]: its an ID

**Pull request merged**